### PR TITLE
Lint check for every file

### PR DESCRIPTION
**Pull request by richtja**
The pylint didn't check every file python in avocado project. It's
happening because pylint assumes that any directory is a package, and
fails if not. This is fixing that and also fix the lint errors which was found by this change.

Reference: #4437
Signed-off-by: Jan Richter <jarichte@redhat.com>

### CODE REVIEW

File: selftests/.data/loader_instrumented/imports.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# docstrings, methods, imports... pylint: disable=C0111,W0403,R0903,C0411,E0611,C0412
+# docstrings, methods, imports... pylint: disable=C0111,R0903,C0411,E0611,C0412
```
Comment (beraldoleal) [inline selftests/.data/loader_instrumented/imports.py:1]: Here you are removing one warning but the commit message says "Lint E0012 fix". Could, please, elaborate a little bit ?

File: selftests/lint.sh [hunk 2]
```diff
@@ -9,22 +9,22 @@ PYLINT=$(which pylint-3 2>/dev/null || which pylint)
 
 # Those are our main packages, we should follow the .pylintrc file with all
 # enabled by default. Some are disabled, we are working to reduce this list.
```
Comment (beraldoleal) [inline selftests/lint.sh:11]: We should update this comment too. replacing  the "packages" word.

**Review by beraldoleal — CHANGES_REQUESTED** [review]
Hi @richtja, well done. Thank you. Besides minor changes, this looks fine.

**Review by willianrampazzo — APPROVED** [review]
Including @beraldoleal's comments. LGTM.

**Comment by willianrampazzo** [comment]
Codecov needs to be fixed: `ImportError: cannot import name 'test_suite' from 'selftests' (/home/travis/build/avocado-framework/avocado/selftests/__init__.py)`

File: selftests/.data/loader_instrumented/imports.py (hunk 1 above)
Comment (richtja) [reply selftests/.data/loader_instrumented/imports.py:1]: Yes the problem is that there are no imports so disabling the `W0403` doesn't make sense. And the `E0012` is complaining about it. `E0012 checks if a bad value for a PyLint option is encountered.` I understand the confusion and I will update the commit message.

File: selftests/lint.sh (hunk 2 above)
Comment (richtja) [reply selftests/lint.sh:11]: I missed that, thanks.

**Comment by richtja** [comment]
Hi @beraldoleal and @willianrampazzo, thanks for your comments, I force-pushed the fixes.

**Review by beraldoleal — APPROVED** [review]
LGTM

**Pull request merged**